applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt_scan_for_broadcast.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/gap.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/sys/byteorder.h>
+
+#include "macros_common.h"
+#include "nrf5340_audio_common.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(bt_mgmt);
+
+/* Any value above 0xFFFFFF is invalid, so we can just use 0xFFFFFFFF to denote
+ * invalid broadcast ID
+ */
+#define INVALID_BROADCAST_ID 0xFFFFFFFF
+#define PA_SYNC_SKIP	     5
+#define SYNC_RETRY_COUNT     6 /* similar to retries for connections */
+
+ZBUS_CHAN_DECLARE(bt_mgmt_chan);
+
+struct bt_le_scan_cb scan_callback;
+static bool cb_registered;
+static char const *srch_name;
+
+struct broadcast_source {
+	char *name;
+	size_t size;
+	uint32_t broadcast_id;
+};
+
+static uint16_t interval_to_sync_timeout(uint16_t interval)
+{
+	uint16_t timeout;
+
+	/* Ensure that the following calculation does not overflow silently */
+	__ASSERT(SYNC_RETRY_COUNT < 10, "SYNC_RETRY_COUNT shall be less than 10");
+
+	/* Add retries and convert to unit in 10's of ms */
+	timeout = ((uint32_t)interval * SYNC_RETRY_COUNT) / 10;
+
+	/* Enforce restraints */
+	timeout = CLAMP(timeout, BT_GAP_PER_ADV_MIN_TIMEOUT, BT_GAP_PER_ADV_MAX_TIMEOUT);
+
+	return timeout;
+}
+
+static void periodic_adv_sync(const struct bt_le_scan_recv_info *info, uint32_t broadcast_id)
```

Comment:
```suggestion
static void periodic_adv_synced(const struct bt_le_scan_recv_info *info, uint32_t broadcast_id)
``` ?

Reply:
No, this is the act of syncing to a periodic advertising